internal/web/templates/pages/package.html

```diff
@@ -259,18 +259,30 @@ <h2 class="text-reset">
                     {{ end }}
                   </ul>
                 </div>
-              {{ else if or (not .Status) .ValidationResult.Requirements }}
-                <div class="alert alert-success m-0 mb-2" role="alert">
-                  <span>The following packages will be installed:</span>
-                  <ul class="mb-0 mt-1">
-                    {{ if not .Status }}
-                      <li>{{ $.Manifest.Name }} ({{ $.SelectedVersion }})</li>
-                    {{ end }}
-                    {{ range .ValidationResult.Requirements }}
-                      <li>{{ .Name }} ({{ .Version }})</li>
-                    {{ end }}
-                  </ul>
-                </div>
+              {{ else }}
+                {{ if or (not .Status) .ValidationResult.Requirements }}
+                  <div class="alert alert-success m-0 mb-2" role="alert">
+                    <span>The following packages will be installed:</span>
+                    <ul class="mb-0 mt-1">
+                      {{ if not .Status }}
+                        <li>{{ $.Manifest.Name }} ({{ $.SelectedVersion }})</li>
+                      {{ end }}
+                      {{ range .ValidationResult.Requirements }}
+                        <li>{{ .Name }} ({{ .Version }})</li>
+                      {{ end }}
+                    </ul>
+                  </div>
+                {{ end }}
+                {{ if and .ValidationResult.Pruned }}
+                  <div class="alert alert-danger m-0 mb-2" role="alert">
```

Comment:
we normally use "danger" for errors and things that are not possible
```suggestion
                  <div class="alert alert-warning m-0 mb-2" role="alert">
```